### PR TITLE
first modification

### PR DESCRIPTION
**Pull request by lubaeisenstein**
This PR updates the file with image

### CODE REVIEW

File: _posts/0000-01-16-lubaeisenstein.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here - new location line 6
```
Comment (beardofedu) [inline _posts/0000-01-16-lubaeisenstein.md]: Maybe add a funny saying related to your image.

File: _posts/0000-01-16-lubaeisenstein.md [hunk 2]
```diff
@@ -3,7 +3,9 @@ layout: slide
...
 
 ![inflatocat](https://octodex.github.com/images/inflatocat.png)
 {: .center}
+
```
Comment (beardofedu) [inline _posts/0000-01-16-lubaeisenstein.md]: you can remove the extra white space

File: _posts/0000-01-16-lubaeisenstein.md [hunk 3]
```diff
@@ -3,7 +3,9 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-16-lubaeisenstein.md]: you could customize the title of your slide

File: _posts/0000-01-16-lubaeisenstein.md (hunk 2 above)
Comment (lubaeisenstein) [reply _posts/0000-01-16-lubaeisenstein.md]: great, thanks

**Pull request merged**